src/densities.jl

```diff
@@ -72,61 +72,11 @@ function compute_density(basis::PlaneWaveBasis, ψ::AbstractVector,
         ρaccu .= 0
         for ik in ikpts
             ρ_k = compute_partial_density(basis, basis.kpoints[ik], ψ[ik], occupation[ik])
-            _symmetrize_ρ!(ρaccu, ρ_k, basis, basis.ksymops[ik], Gs)
+            # accumulates all the symops of ρ_k into ρaccu
+            _symmetrize!(ρaccu, ρ_k, basis, basis.ksymops[ik], Gs)
```

Comment:
Yeah I agree. The thing is that this function is pretty internal because it accumulates rather than erases, as would be more conventional with a `symmetrize!`, and doesn't normalize, as you would expect from a symmetrize function. Let's just rename it to `add_all_symops` or something?

Reply:
accumulate_over_symops! is better

Reply:
Agreed